utils.py

```diff
@@ -29,9 +29,7 @@ def write_file():
         write_file()
 
 
-def read_config(section, setting):
-    if not section:
-        section = "kobo_backup_config"
+def read_config(setting, section="kobo_backup_config"):
```

Comment:
default section to read from the config file is kobo_backup_config unless otherwise specified.